src/store/ChartStore.js

```diff
@@ -519,6 +519,14 @@ class ChartStore {
             this.stxx = null;
         }
     }
+
+    @action.bound onHistorical() {
+        this.contextPromise.then(() => {
+            // console.log('asdas');
```

Comment:
dude. Please check your code (every single line) before you send for review

Reply:
so sorry